src/mds/MDSDaemon.cc

```diff
@@ -339,6 +346,7 @@ void MDSDaemon::clean_up_admin_socket()
   admin_socket->unregister_command("flush_path");
   admin_socket->unregister_command("export dir");
   admin_socket->unregister_command("dump cache");
+  admin_socket->unregister_command("dump below");
```

Comment:
(Subjective) Maybe "dump path" or "dump tree" would be clearer than "dump below"?